assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/DeletePriceModifierButton.tsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
-import { DeleteOutlined } from '@ant-design/icons';
+import React, { useCallback } from 'react';
+import { __ } from '@wordpress/i18n';
 
-import { EspressoButton } from '@application/ui/input';
+import { ConfirmDelete } from '@appLayout/confirmDelete';
+import { EspressoButton, Icon } from '@application/ui/input';
 import { PriceModifierProps } from '../types';
 import { useDataState } from '../data';
 
 const DeletePriceModifierButton: React.FC<PriceModifierProps> = ({ price }) => {
 	const { deletePrice } = useDataState();
+	const onClick = useCallback(() => deletePrice(price.id, price.isNew), [price.id, price.isNew]);
 
-	const onClick = () => deletePrice(price.id, price.isNew);
-
-	return <EspressoButton icon={<DeleteOutlined />} onClick={onClick} />;
+	return (
+		<ConfirmDelete
+			children={<EspressoButton icon={Icon.TRASH} tooltip={__('delete price modifier')} />}
```

Comment:
have gone this way because of this:

![image](https://user-images.githubusercontent.com/1477453/78241977-eff16c80-74e9-11ea-9514-5c5f79f8b53a.png)